src/ent/bit_rack.h

```diff
@@ -50,16 +63,49 @@ bool bit_rack_is_compatible_with_ld(const LetterDistribution *ld) {
          max_letter_count_on_board <= bit_rack_max_letter_count;
 }
 
-BitRack bit_rack_create_empty(void) {
+static inline bool bit_rack_type_has_expected_size(void) {
+  return sizeof(BitRack) == 16;
+}
+
+static inline bool bit_rack_is_compatible_with_endianness(void) {
+#if defined(__APPLE__)
+  return OSHostByteOrder() == OSLittleEndian;
+#else
+  return __BYTE_ORDER == __LITTLE_ENDIAN;
+#endif
+}
```

Comment:
Same as above. Prefer to handle in compat/endian_conv.h

Reply:
Removed this, this check no longer exists, compatible with either endianness (though big endian is untested)